autoload/editorconfig_core.vim

```diff
@@ -0,0 +1,138 @@
+" autoload/editorconfig_core.vim: top-level functions for
+" editorconfig-core-vimscript.
+" Copyright (c) 2018 Chris White.  All rights reserved.
```

Comment:
Can you add a license header block for every file? You can refer to other files in the plugin for how to do it. Also, should the year be 2019?